app/webapp/src/main/java/uk/ac/ebi/fg/annotare2/web/gwt/common/client/view/DataFilesUploadViewImpl.java

```diff
@@ -418,17 +418,25 @@ public void onFileAdded(ResumableUploader uploader, ResumableFile file) {
         public void onFilesAdded(ResumableUploader uploader, JsArray<ResumableFile> files) {
             boolean shouldUpload = true;
             boolean fileBlocked = false;
+            boolean invalidFileName = false;
             StringBuilder sb = new StringBuilder();
             sb.append("The file(s) already exist.<br/>To re-upload, please delete and upload again.<br/><br/>"); //<br/> added here because Notification panel display this as HTML so simple new line character won't work.
 
             StringBuilder blockedFiles = new StringBuilder();
             blockedFiles.append("File extension(s) not allowed. <br/> Please upload again with correct file format.<br/><br/>");
 
+            StringBuilder invalidFileNameMsg = new StringBuilder();
```

Comment:
I think this `StringBuilder` can be removed as you are not appending anything after the initial text. You can directly use the string text in `NotificationPopupPanel`